### ISSUE TITLE
[Order Creation] Accessibility → Products Total and Order Total should be grouped with the value

### ISSUE THREAD

**Issue by shiki** [woocommerce/woocommerce-android#5812]
When navigating to the "Products total" using TalkBack, it should be read along with the value. Having the value and the label read together helps minimize the number of accessible elements on the screen. This makes it easier to navigate using TalkBack. 

The same could be done with "Order total". 

<img src="https://user-images.githubusercontent.com/198826/152882006-250a8c6f-7fa3-4e48-9489-216735ec9d36.png" width="300">
